### PR TITLE
Added client version field.

### PR DESCRIPTION
**Pull request by illiriks**
Resolves issue # 46.

### CODE REVIEW

File: lib/client.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ var superagent = require("superagent");
 var errors = require("./errors");
 var request = require("request");
 
+var SDK_VERSION_NUMBER = "1.3.1";
```
Comment (dtolb) [inline lib/client.js]: Pull this from the package.json .

We don't want to have to update this value in two places... too easy to get out of sync. 

Check here how we do this in node v2.

[How to pull from package.json](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/client.js#L4)

File: lib/client.js [hunk 2]
```diff
@@ -60,10 +62,13 @@ function Client(userId, apiToken, apiSecret, options) {
...
 		return request({
-			url  : this.prepareUrl(uri),
-			auth : {
+			url     : this.prepareUrl(uri),
+			auth    : {
 				user : apiToken,
 				pass : apiSecret
+			},
+			headers : {
+				"user-agent" : "node-bandwidth-v" + SDK_VERSION_NUMBER
```
Comment (dtolb) [inline lib/client.js]: [Check here](https://github.com/bandwidthcom/node-bandwidth/blob/2.0-preview/lib/client.js#L48)

For how we did this is version 2. I think they're both about the same and no preference.

**Comment by dtolb** [comment]
Also, we should verify value is set correctly in the tests.

I think probably in [`../test/client.js`](https://github.com/bandwidthcom/node-bandwidth/blob/master/test/client.js)

**Comment by illiriks** [comment]
Updated tests. I tested by adding to `test/helper.js` a stub to make a `nock` that expected a user agent header, and then adding a test to `test/client.js` that made a call using that `nock`. Alternatively, this functionality could be tested by requiring it on _every_ `nock`; if this is preferable, I can make those changes.

File: test/client.js [hunk 3]
```diff
@@ -91,11 +91,17 @@ describe("client tests", function () {
...
-		it("should  handle requests without output", function (done) {
+
+		it("should handle requests without output", function (done) {
 			helper.nock().get("/v1/test").reply(200);
 			client.makeRequest("get", "/test", done);
 		});
 
+		it("should provide a user agent header", function (done) {
+			helper.nockRequiringUserAgent().get("/v1/test").reply(200);
+			client.makeRequest("get", "/test", done);
```
Comment (dtolb) [inline test/client.js:102]: there should be an assertion here somewhere? Right?

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "node-bandwidth",
-  "version": "1.3.0",
+  "version": "1.3.1",
```
Comment (dtolb) [inline package.json:3]: We update version in a release branch, no issue here, just letting ya know

**Pull request merged**